### DISCUSSION TITLE
Hello!

### DISCUSSION THREAD

**Discussion by sampsyo** [beetbox/beets#3806] [General]
Just giving Discussions a shot. If we like it, maybe we'll switch over from the venerable old [Discourse](https://discourse.beets.io)!

**Comment by jef** [comment]
Can't wait! Hopefully we'll get some good questions!

**Comment by rugk** [comment]
What do you have down on Discourse?

**Reply by jef** [reply]
If you're asking what we have against Discourse: then it's nothing! Nothing wrong with Discourse. This is an alternative to it in which users can discuss without have another account and things can be some what unified.

Another bonus is that maintainers can create issues directly from GitHub discussions, so it makes that portion easier for us.

[Homebrew](https://github.com/Homebrew/discussions) and others have also deprecated their Discourse in favor of GitHub Discussions and thought we could do something similar if users start to use it.

**Reply by rnkn** [reply]
Also doesn't Discourse have quite steep requirements for hosting?

**Reply by jef** [reply]
Since this an open source organization, I believe we get to use Discourse for free. Otherwise, yes, would definitely be expensive!
